2023/gaussian-convolution/spanish/sentence_translations.json

```diff
@@ -650,7 +650,7 @@
  {
   "input": "Almost always you end up with a completely different kind of function, but here there's a sort of stability to the process.",
   "model": "nmt",
-  "translatedText": "Casi siempre terminas con un tipo de función completamente diferente, pero aquí hay una especie de estabilidad en el proceso.",
+  "translatedText": "Casi siempre terminas con un tipo de función completamente distinta, pero aquí hay una especie de estabilidad en el proceso.",
```

Comment:
I am curious about this. To me "diferente" and "distinta" mean pretty much the same. "Diferente" probably sounds a bit more "natural" to me in this context (natural as in I would say it that way, both words pretty much convey the same meaning, do they not?).

Reply:
That's probably more personal than anything (or even a localization) for me `distinta` feels more natural.

Reply:
Well, thinking a bit about it there is an issue, it should be `distinto` since the associated noun is `tipo` which is masculine, but this does not sound right. I think that we should keep it as `diferente`.

Reply:
Well, thinking a bit about it there is an issue, it should be `distinto` since the associated noun is `tipo` which is masculine, but this does not sound right. I think that we should keep it as `diferente`.

Reply:
We should keep "diferente"

Reply:
Suggestion committed [here](https://github.com/pabloasenjo3/captions/commit/aeee5157da96b74b22c8b4bce0020cc198d9ae02).